[en]: Add PATH note for install macos kubectl (#28418)
kubernetes/website: content/en/docs/tasks/tools/install-kubectl-macos.md

   sudo chown root: /usr/local/bin/kubectl
   ```

   {{< note >}}

>    On macOS,  `/usr/local/bin/` is usually included in your `$PATH`.

Reviewer: How about "Make sure `/usr/local/bin` is in your PATH environment variable."

Author: That pretty good👍

   sudo chown root: /usr/local/bin/kubectl
   ```

   {{< note >}}

>    For more information about what PATH is, please refer to this [article](https://kb.iu.edu/d/acar).

Reviewer 2: I'd write:
```suggestion
   For more information about what PATH is, refer to [Set or modify a path in Unix](https://kb.iu.edu/d/acar).
```
or just omit that line.

>    On MacOS, usually `/usr/local/bin/` will be included in the PATH.

Reviewer 2: ```suggestion
   On macOS,  `/usr/local/bin/` is usually included in your `$PATH`.
```
?

   You can run the following command to detect:
   ```bash
   es=$(echo $PATH | grep /usr/local/bin);if [[ -z "$es" ]]; then;echo "Detected that PATH does not contain /usr/local/bin";echo "Please refer to https://kb.iu.edu/d/acar, add /usr/local/bin to the PATH";exit 255;fi; echo "You can proceed to the next step";

>    ```

Reviewer 2: This might close the terminal, surprising a reader. Let's give them a command to just _check_ it, that never has side-effects.

>    ```

Reviewer: This is an overkill. 
Just mention that users must have `/usr/local/bin` must be in their `$PATH` would be okay.
Adding scripts may introduce unnecessary problems, considering that not all users use Bash.
All new contents, once added, are to be maintained. Please bear that in mind when proposing fixes.

> For more information about what PATH is, refer to [Set or modify a path in Unix](https://kb.iu.edu/d/acar).

This is beyond what kubernetes website is about.

Author: TIL.
